server/monitor_test.go

```diff
@@ -3791,6 +3793,10 @@ func TestMonitorAccountz(t *testing.T) {
 		t.Fatalf("Body missing value. Contains: %s", body)
 	} else if !strings.Contains(body, `"subscriptions": 36,`) {
 		t.Fatalf("Body missing value. Contains: %s", body)
+	} else if !strings.Contains(body, `"is_system_account": true,`) {
```

Comment:
Would have to change that if you change the json tag name.

Reply:
fixed